### DISCUSSION TITLE
How to know the cause of an unexpected stop of an app

### DISCUSSION THREAD

**Discussion by xiaofanku** [aio-libs/aiohttp#9035] [General]
Write a live streaming program with aiohttp and aiortc. It is found that sometimes the app stops unexpectedly shortly after startup. Streaming Server Console Output: closed: read tcp 192.168.3.14:1935->192.168.3.14:57440: wsarecv: An existing connection was forcibly closed by the remote host.

on_shutdown registered listening function is also not executed. Where can I find out what's causing the app to stop?

**Comment by Dreamsorcerer** [comment]
Show some code, I have no how that message relates to aiohttp..

**Reply by xiaofanku** [reply]
> Show some code, I have no how that message relates to aiohttp..

I mainly want to know how to configure aiohttp to see the error message when this happens

**Comment by xiaofanku** [comment]
@Dreamsorcerer thanks your comment. I seem to have found the root of the problem. Now you can be sure that it has nothing to do with aiohttp
